docs/getting_started_guide.md

```diff
@@ -0,0 +1,248 @@
+# Contributing
+
+A general overview of our preferred Git Workflow can be found here:
+
+[https://gist.github.com/joshdmiller/dd1a0cc40e06f1aa47c7](https://gist.github.com/joshdmiller/dd1a0cc40e06f1aa47c7)
+
+One key point is that we would prefer all Pull Requests be submitted from branches on our main `StoryShop` repos, rather than from engineers' forks, so that we are in a better position to expedite review & approval by making small corrections if need be. If you are not already a member of the org, please let us know if you would like to be granted write access for the purpose of submitting a PR.
+
+## Introduction
+
+For this walkthrough, we're going to create a new route that handles the items for a character. This is not something that we would implement in the actual project and is only intended to demonstrate the structure of the app.
+
+A few things you should review before getting started:
+
+1. MaterialUI
+2. Tape
+3. RxJS Observables
+4. React Stamp
+
+## Routing
+
+The directory structure mirrors the URL structure. For our new `items` route, we would need to nest it within a single character.
+
+```js
+// src/routes/worlds/one/characters/one/items/route.js
+import React from 'react';
+import Items from './';
+
+export default {
+  path: ':character_id/items',
+  component: Items,
+};
+```
+
+And since this is a child route of `characters`, we should add this as a childRoute:
+
+```js
+// src/routes/worlds/one/characters/index.js
+import uiStore from 'stores/ui';
+import { setTheme } from 'stores/actions/meta';
+import characterListRoute from './list/route';
+import characterRoute from './one/route';
+import itemsRoute from './one/items/route';
+
+export default {
+  path: 'characters',
+  indexRoute: characterListRoute,
+  onEnter: () => {
+    uiStore.dispatch( setTheme( 'characters' ) );
+  },
+  childRoutes: [
+    characterRoute,
+    itemsRoute,
+  ],
+};
+```
+
+The last thing we need to do for routing is update our route utility, which keeps track of all our routes.
+
+```js
+// src/utils/paths.js
+...
+
+export const characterItems = ( wid, cid ) => `/app/worlds/${wid}/characters/${cid}/items`;
+
+...
+```
+
+## Components
+
+We're going to build a simple component that renders a list of items that belong to the character.
+
+Since we're using MaterialUI, we will use their components whenever possible. In this case, we are using the `List` and `ListItem` components from MaterialUI.
+
+```js
+// src/../characters/items/index.js
```

Comment:
These `..` are confusing because they have semantic meaning in file paths. I think the only missing part of the path is "components", right?